definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -26,5 +26,19 @@
     "CaseEventID": "testMoveCaseFromAwaitingServicePaymentToAwaitingServiceConsideration",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "26/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "testAosDrafted",
```

Comment:
I understand it's not to be released. Ever?

Reply:
Yes, not to be released ever. interestingly the `testAosStarted` and `testAosAwaiting` are already in the core file. Not sure why